docs/project/2024/abdul.mirghani/index.md

```diff
@@ -0,0 +1,120 @@
+# Project Name
+Object Detection and Alert System
+:::info 
+
+**Author**: Mahmoud Mirghani Abdelrahman
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-Abd210
+
+:::
+
+## Description
+
+The project utilizes an HC-SR04 ultrasonic sensor, a servo motor ,an lcd, and a buzzer. Here's how it works:
+
+Radar-like Functionality: The servo motor is attached to the ultrasonic sensor, allowing it to sweep back and forth like a radar. This setup helps the sensor detect objects across a range of angles.
+
+Detection and Alerts: When an object is within range, the ultrasonic sensor measures the distance to the object and stops the servo motor's rotation. A buzzer then sounds, alerting me to the presence of an obstacle.
+
+Real-time Feedback: The buzzer continues to sound until the object moves away, allowing the servo to resume its sweeping motion. During detection, the system displays the distance to the object and the servo's angle on an LCD screen, providing me with real-time information about the object's position.
+
+This project effectively combines ultrasonic sensing, servo control, and audio alerts to create a functional radar-like detection mechanism with real-time feedback, making it suitable for obstacle detection and range measurement applications.
+
+## Motivation
+
+I chose this project to create a functional radar-like system for obstacle detection and range measurement, utilizing my skills in ultrasonic sensing, servo control, and real-time feedback. I was also inspired by a YouTube video on similar technology.
+
+## Architecture 
+    Components:
+Sensor Module: The HC-SR04 ultrasonic sensor measures the distance to nearby objects.
+
+Control Module: 
+    A Raspberry Pi Pico microcontroller manages the sensor's operations, controlling the servo motor and processing data from the ultrasonic sensor.
+
+Alert Module:
+    A buzzer provides an audio alert when an object is detected.
+
+Display Module:
+    An LCD screen provides real-time feedback, displaying the distance to the detected object and the angle of the servo motor.
+
+    Connections:
+LCD Screen: 
+    Connected to the Pico with:
+        SDA to PIN_16
+        SCL to PIN_17
+        Ground to Ground
+        VSYS to VDD
+Ultrasonic Sensor:
+    Connected to the Pico with:
+        Trig to GP21
+        Echo to GP20
+        VCC to VBUS
+        Ground to Ground
+Buzzer:
+    Connected to the Pico with:
+        Ground to Ground
+        Positive to PIN2
+Servo Motor:
+    Connected to the Pico with:
+        Signal to PIN28
+        VCC to VBUS
+        Ground to Ground
```

Comment:
````suggestion
```
LCD Screen: 
    Connected to the Pico with:
        SDA to PIN_16
        SCL to PIN_17
        Ground to Ground
        VSYS to VDD
Ultrasonic Sensor:
    Connected to the Pico with:
        Trig to GP21
        Echo to GP20
        VCC to VBUS
        Ground to Ground
Buzzer:
    Connected to the Pico with:
        Ground to Ground
        Positive to PIN2
Servo Motor:
    Connected to the Pico with:
        Signal to PIN28
        VCC to VBUS
        Ground to Ground
```
````